docs/src/main/paradox/docs/delta/api/statistics-api.md

```diff
@@ -0,0 +1,328 @@
+They provide ways to get insights about the data and their relationships in terms of types (@type field).
+  
+
+@@@ note { .tip title="Authorization notes" }	
+
+When reading statistics, the caller must have `resources/read` permissions on the current path of the project or the 
+ancestor paths.
+
+Please visit @ref:[Authentication & authorization](authentication.md) section to learn more about it.
+
+@@@
+
+@@@ note { .warning }
+
+The described endpoints are experimental and the responses structure might change in the future.
+
+@@@
+
+## Fetch relationships
+
+Obtains all the @type relationships and their counts - defined by nodes and edges - existing on a project.
+
+The edges are the properties linking different nodes, and the nodes are the resources containing a certain @type.
+```
+GET /v1/statistics/{org_label}/{project_label}/relationships
+```
+
+**Example**
+
+Request
+:   @@snip [fetch-relationships.sh](assets/statistics/fetch-relationships.sh)
+
+Response
+:   @@snip [fetched-relationships.json](assets/statistics/fetched-relationships.json)
+
+
+## Fetch properties
+
+Obtains all the @type properties and their counts. 
+
+The different between properties and relationships is
+that properties are enclosed inside the same resource, while relationships are statements between different resources.
+```
+GET /v1/statistics/{org_label}/{project_label}/properties/{type}
+```
+
+...where `{type}` is an IRI defining for which @type we want to retrieve the properties.
+
+**Example**
+
+Request
+:   @@snip [fetch-properties.sh](assets/statistics/fetch-properties.sh)
+
+Response
+:   @@snip [fetched-properties.json](assets/statistics/fetched-properties.json)
+
+## Fetch progress
+
+```
+GET /v1/statistics/{org_label}/{project_label}/progress
+```
+It returns:
+
+- the dateTime of the latest consumed event (`lastProcessedEventDateTime`).
+- the number of consumed events (`eventsCount`).
+- the number of consumed resources (`resourcesCount`). A resource might be made of multiple events (create, update, deprecate), so this number will always be smaller or equal to `eventsCount`.
+
+**Example**
+
+Request
+:   @@snip [fetch-progress.sh](assets/statistics/fetch-progress.sh)
+
+Response
+:   @@snip [fetched-progress.json](assets/statistics/fetched-progress.json)
+
+## Internals
+
+In order to implement the described endpoints we needed a way to transform our data so that it would answer the desired questions in a performant manner.
+
+The proposed solution was to stream our data, transform it and push it to a dedicated ElasticSearch index (one index per project). Then at query time we can run [term aggregations](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html) in order to get the desired counts.
+
+### Document structure
+
+An example of the ElasticSearch Document looks as follows:
+
+```json
```

Comment:
This is a search response, just the source of a single document should be enough